CHANGELOG.md

```diff
@@ -138,6 +138,10 @@
   [nickgravelyn](https://github.com/nickgravelyn)
   [#757](https://github.com/CocoaPods/Xcodeproj/pull/757)
 
+* Fix crash when adding sub project.
```

Comment:
nit: please add two empty spaces after the period for markdown formatting.

also add your name for credit!

Reply:
Fixed in update.